Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml

```diff
@@ -0,0 +1,384 @@
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: 63ee4a95-5502-4559-888c-b9db5e4078db
+    type: regular
+    task:
+      id: 63ee4a95-5502-4559-888c-b9db5e4078db
+      version: -1
+      name: Prisma Access - Get Service Connection Status
+      script: '|||prisma-access-query'
+      type: regular
+      iscommand: true
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      limit: {}
+      query:
+        simple: action getServiceConnectionRegionalStat
+      querystring:
+        simple: query action getServiceConnectionRegionalStat
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 515
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "2":
+    id: "2"
+    taskid: d3bac10a-bc2b-4663-8ab9-a5bc9dd93c83
+    type: regular
+    task:
+      id: d3bac10a-bc2b-4663-8ab9-a5bc9dd93c83
+      version: -1
+      name: Prisma Access - Get Firewall Status
+      script: '|||prisma-access-query'
+      type: regular
+      iscommand: true
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      query:
+        simple: action getFWaaSRegionalStat
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 515
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: c5ef9d40-3bc0-4888-8780-59e4b44cbd9d
+    type: condition
+    task:
+      id: c5ef9d40-3bc0-4888-8780-59e4b44cbd9d
+      version: -1
+      name: Any tunnels down?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEqualString
+          left:
+            value:
+              simple: PrismaAccess.QueryResults.Status.value
+            iscontext: true
+          right:
+            value:
+              simple: OK
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 5bdbd75e-ac46-4824-83c8-91af11efffbf
+    type: regular
+    task:
+      id: 5bdbd75e-ac46-4824-83c8-91af11efffbf
+      version: -1
+      name: Slack - Send Alert
+      description: Sends a message to a user, group, or channel.
+      script: SlackV2|||send-notification
+      type: regular
+      iscommand: true
+      brand: SlackV2
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      blocks: {}
+      channel:
+        simple: netops
```

Comment:
A customer might not have "netops" channel and right now it is hardcoded, which will cause the playbpook to fail for the majority of customers. We can add this as playbook input instead. 

Also, what happens if the customer does not even use slack? Slack should be optional. Add a check if integration is enabled and a channel was provided as playbook input.

Reply:
@michalgold Removing Slack step